source/dubregistry/mongodb.d

```diff
@@ -6,27 +6,56 @@ import vibe.db.mongo.mongo : connectMongoDB;
 import vibe.db.mongo.settings : MongoClientSettings, MongoAuthMechanism, parseMongoDBUrl;
 import std.typecons : Nullable;
 
+/// Database name
 string databaseName = "vpmreg";
 private Nullable!MongoClientSettings _mongoSettings;
 
 @safe:
 
-MongoClientSettings mongoSettings() {
+/**
+Initializes mongoSettings if null
+
+Params:
+	allowUnauthenticated = whether to allow unauthenticated access
+
+Examples:
+	dub -- --allow-unauthenticated=true
```

Comment:
It's not really an example for this function

Reply:
I have no idea what I was thinking at that moment. Probably haven't had my coffee 😄